Arduino/libraries/Capacitive_Fingerprint_Reader/finger.cpp

```diff
@@ -58,16 +54,16 @@ uint8_t TxAndRxCmd(uint8_t Scnt, uint8_t Rcnt, uint16_t Nms)
 	 
 	 memset(finger_RxBuf,0,sizeof(finger_RxBuf));   ////////
  
-	 mySerial.flush();  /////
+	 serialFlush();  /////
```

Comment:
Had to create a new serialFlush function due to how Serial2.flush() worked. Currently Serial2.flush() waits for serial transmission to complete. We need the version that clears the serial buffer.